thoth/glyph/formatter.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# thoth-glyph
+# Copyright(C) 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Generate CHANGELOG entries out of commit messages using AI/ML techniques."""
```

Comment:
Usually, this holds module (`thoth.glyph.formatter` docstring - info about what's present in this file). Same for other files newly introduced.

Reply:
Fixed. Docstrings now contain file-specific info.